packages/testcontainers/src/container-runtime/clients/container/container-client.ts

```diff
@@ -22,7 +22,11 @@ export interface ContainerClient {
   stop(container: Container, opts?: { timeout: number }): Promise<void>;
   attach(container: Container): Promise<Readable>;
   logs(container: Container, opts?: ContainerLogsOptions): Promise<Readable>;
-  exec(container: Container, command: string[], opts?: { log: boolean }): Promise<ExecResult>;
+  exec(
+    container: Container,
+    command: string[],
+    opts?: Partial<ExecOptions>,
+  ): Promise<ExecResult>;
```

Comment:
It's me, or probably you don't have configured prettier to format on save? If you run `npm run format` you can see how there are at least 4 files to be formatted.

We should add to our precommit hook the format script adapted to lint-stage? 🤔

Reply:
I just executed `npm run format` and pushed the changes 😁

----

> We should add to our precommit hook the format script adapted to lint-stage? 🤔

I would add the format task (before running the linter) in the pre-commit git hook.  But this is a personal decision. This is something someone who works frequently with this codebase should take. cc @cristianrgreco